modules/cli/src/main/scala/scala/cli/commands/shared/ScalaJsOptions.scala

```diff
@@ -37,44 +37,58 @@ final case class ScalaJsOptions(
 
   @Group(HelpGroup.ScalaJs.toString)
   @Tag(tags.should)
-  jsCheckIr: Option[Boolean] = None,
+    jsCheckIr: Option[Boolean] = None,
 
   @Group(HelpGroup.ScalaJs.toString)
   @HelpMessage("Emit source maps")
   @Tag(tags.should)
     jsEmitSourceMaps: Boolean = false,
+
   @Group(HelpGroup.ScalaJs.toString)
   @HelpMessage("Set the destination path of source maps")
   @Tag(tags.should)
    jsSourceMapsPath: Option[String] = None,
+
+  @Group(HelpGroup.ScalaJs.toString)
+  @HelpMessage("A file relative to the root directory containing import maps for ES module imports")
+  @Tag(tags.experimental)
```

Comment:
Should be fine to tag this as restricted
```suggestion
  @Tag(tags.restricted)
```

Reply:
agreed